styles/style.css

```diff
@@ -88,33 +103,44 @@ body {
 	max-width: 100px;
 	background: #ce5f31;
 }
-
+.container {
+	flex: 1 1 30%;
```

Comment:
You cannot write your own `container` class when using Bootstrap. It will interfere with the grid.